modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/FDSPropsTransformer/FDSDataRenderers/ObjectEntryStatusDataRenderer.tsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import {ClayTooltipProvider} from '@clayui/tooltip';
+import React, {useEffect, useState} from 'react';
+
+import './ObjectEntryStatusDataRenderer.scss';
+
+import {fetch, sub} from 'frontend-js-web';
+
+function formatDisplayDate(value: Date) {
+	const date = new Date(value);
+
+	const day = date?.getDate();
+
+	const hoursAndMinutes = date?.toLocaleTimeString('default', {
+		hour: '2-digit',
+		minute: '2-digit',
+	});
+
+	const monthName = date?.toLocaleDateString('default', {month: 'short'});
+
+	const year = date?.getFullYear();
+
+	return `${monthName} ${day}, ${year} ${hoursAndMinutes}`;
```

Comment:
We have already had a great improvement in the return.

I believe we don't need to go into too much detail, we can return something like:

`return ${month} ${day}, ${year} ${hours};`

Reply:
done!